### PR TITLE
Fail redis and mongo tests if incorrect URI provided + some storages tests refactoring

### PR DESCRIPTION
**Pull request by Rishat-F**
# Description

If incorrect URIs provided to "**--redis**" and/or "**--mongo**" options tests should fail with **errors** instead of **skipping**.
Otherwise the next scenario is possible:

1. developer breaks RedisStorage and/or MongoStorage code
2. tests are run with incorrect redis and/or mongo URIsprovided by "**--redis**" and "**--mongo**" options (for example, wrong port specified)
3. tests pass because skipping doesn't fail tests run
4. developer or reviewer doesn't notice that redis and/or mongo tests were skipped
5. broken code gets in codebase

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by Olegt0rr — CHANGES_REQUESTED** [review]
LGTM, but make 100% coverage, please

File: aiogram/fsm/storage/mongo.py [hunk 1]
```diff
@@ -39,7 +39,10 @@ def __init__(
 
     @classmethod
     def from_url(
-        cls, url: str, connection_kwargs: Optional[Dict[str, Any]] = None, **kwargs: Any
+        cls,
+        url: str,
+        connection_kwargs: Optional[Dict[str, Any]] = {"serverSelectionTimeoutMS": 2000},
```
Comment (Olegt0rr) [inline aiogram/fsm/storage/mongo.py:44]: Why you pass this values as default?

Comment (Rishat-F) [reply aiogram/fsm/storage/mongo.py:44]: Yes, I agree. That was bad idea to do like this.

**Comment by Rishat-F** [comment]
> LGTM, but make 100% coverage, please

Done

File: examples/multibot.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 from typing import Any, Dict, Union
 
 from aiohttp import web
+from finite_state_machine import form_router
```
Comment (Olegt0rr) [inline examples/multibot.py:7]: Looks like it's conflicts with command `make reformat` from contribution guide

File: tests/conftest.py [hunk 3]
```diff
@@ -5,19 +5,27 @@
...
+from aiogram.fsm.storage.redis import RedisStorage
 from tests.mocked_bot import MockedBot
 
 DATA_DIR = Path(__file__).parent / "data"
 
+CHAT_ID = -42
+USER_ID = 42
+
+skip_message_pattern = 'Need "--{db}" option with {db} URI to run'
+invalid_uri_pattern = "Invalid {db} URI {uri!r}: {err}"
```
Comment (Olegt0rr) [inline tests/conftest.py:27]: Looks like CONSTANT vars

**Review by Olegt0rr — CHANGES_REQUESTED** [review]

File: tests/conftest.py (hunk 3 above)
Comment (Rishat-F) [reply tests/conftest.py]: Done

File: examples/multibot.py (hunk 2 above)
Comment (Rishat-F) [reply examples/multibot.py:7]: Nothing had changed after running `make reformat`

Comment (Olegt0rr) [reply examples/multibot.py:7]: Please update and merge `aiogram:dev-3.x` to this branch.
We're just updated dependencies of the format checkers and linters.

Comment (Rishat-F) [reply examples/multibot.py:7]: Done

**Review by Olegt0rr — APPROVED** [review]

**Review by JrooTJunior — APPROVED** [review]
Thanks for your contribution.

I will update the changelog message and category, this change does not affect the functionality of the framework, but improves the development process of the framework itself, so these changes should be marked as "misc" and not as "bugfixes".

**Pull request merged**